HIAPI/Models/User.swift

```diff
@@ -14,10 +14,7 @@ import APIManager
 import Foundation
 
 public struct User: Codable, APIReturnable {
-    public let id: String
-    public let username: String
-    public let firstName: String
-    public let lastName: String
+    public let userId: String
```

Comment:
why we not storing `name` here as well?

Reply:
User still has `name` but only the Profile's `displayName` will be displayed in the app!